framework/src/st_things/things_stack/framework/things_security_manager.c

```diff
@@ -829,15 +832,17 @@ static OCStackResult save_signed_asymmetric_key(OicUuid_t *subject_uuid)
 
 #ifdef USE_SSS
 	if (dm_get_easy_setup_use_artik_crt()) {
-		if (things_sss_key_handler_init() < 0) {
+		if (!g_b_init_key && things_sss_key_handler_init() < 0) {
```

Comment:
This is error log, log should include information about g_b_init_key 's state & return value of things_sss_key_handler_init()

Reply:
hi @Taejun-Kwon , should I put this in logs `THINGS_LOG_E(TAG, "g_b_init_key = 0 and InitializeSSSKeyHandlers() Fail");`  and  same thing for g_b_init-cert ?